assignments/02-airbnb-layout/css/style.css

```
@@ -1,3 +1,42 @@
 h1 {
-  color: salmon;
+    color: salmon;
 }
+
+body {
+    background-color: gainsboro;
+}
+
+.headline {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+}
+
+.flex-div {
+    display: flex;
+    justify-content: space-between;
+}
+
+img {
+    width: 300px;
+    height: 300px;
+}
+
+.cities.img {
```

Select only `.cities`
```suggestion
.cities
```